### PR TITLE
Send Register Dog form to mongoDB

### PR DESCRIPTION
**Pull request by lajanzen**
- Implement useState 
- Implement postUser function and add UserDog Type

### CODE REVIEW

File: src/app/pages/RegisterDog/RegisterDog.tsx [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
 import LabeledInput from "../../components/LabeledInput/LabeledInput";
 import styles from "./RegisterDog.module.css";
 
 function RegisterDog(): JSX.Element {
+  const history = useHistory();
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [city, setCity] = useState("");
+  const [birth, setBirth] = useState(0);
```
Comment (lmachens) [inline src/app/pages/RegisterDog/RegisterDog.tsx]: Is this the year of birth? 
I think you could clarify it.

**Review by lmachens — APPROVED** [review]
I like that you have two kinds of users :)

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

File: src/app/pages/RegisterDog/RegisterDog.tsx (hunk 1 above)
Comment (lajanzen) [reply src/app/pages/RegisterDog/RegisterDog.tsx]: Thanks, will do! :)

**Pull request merged**